extracted/konan.serializer/src/org/jetbrains/kotlin/descriptors/konan/KonanModuleDescriptorUtils.kt

```diff
@@ -4,6 +4,7 @@ import org.jetbrains.kotlin.descriptors.ModuleDescriptor
 import org.jetbrains.kotlin.konan.library.KONAN_STDLIB_NAME
 import org.jetbrains.kotlin.name.Name
 
-private val STDLIB_MODULE_NAME = Name.special("<$KONAN_STDLIB_NAME>")
+//TODO: this is should be reverted!!!
+private val STDLIB_MODULE_NAME = Name.special("<stdlib>")
```

Comment:
Why it is needed? @alexander-gorshenev could you suggest better solution?

Reply:
because here compiler fails with: "KonanModuleDescriptorUtils.kt: (7, 50): Overload resolution ambiguity: 
public const val KONAN_STDLIB_NAME: String defined in org.jetbrains.kotlin.konan.library
public const val KONAN_STDLIB_NAME: String defined in org.jetbrains.kotlin.konan.library"

Reply:
The proper fix would require coordinated changes in Kotlin and Kotlin/Native. Let's have it like that for now and deal with the proper  little later.
@vvlevchenko we could make it
KONAN_STDLIB_MODULE_NAME and KONAN_STDLIB_LIBRARY_NAME, 
just to get rid of name clash in composite builds for now.